### ISSUE TITLE
Talea won't arrpegiate

### ISSUE THREAD

**Issue by jmcymaticsounds** [jeremymuller/Sha-Bang-Modules#20]
I watched the demo video and I love the poly rhythm idea! Unfortunately, I can't get more than one pitch even when holding down multiple keys. I've tried with my computer keyboard and an external midi keyboard with no luck either way. When I activate the octave button I hear multiple octaves of a pitch, but still only of one note.

**Comment by Petervos2018** [comment]
Did you activate more than 1 note ?

![afbeelding](https://user-images.githubusercontent.com/46076474/107231744-f103fc00-6a20-11eb-8625-1c087d9fcb55.png)

**Comment by jeremymuller** [comment]
Yes, in reference to @Petervos2018 response, I forgot to mention in the video that your MIDI needs to be polyphonic. But did this fix your problem, @jmcymaticsounds?

**Comment by jmcymaticsounds** [comment]
Yes, this fixed my issue. Just found out about this amazing software last week, please excuse the noob question, lol.
